setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = T001
```

Comment:
I use [flake8-print](https://pypi.python.org/pypi/flake8-print) in my standard config, which is not a useful configuration for a CLI; this line disables print checking.